frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -96,23 +96,15 @@ class Transaction extends Component<Props, State> {
     }
 
     private focusEdit = () => {
-        if (this.editButton) {
-            this.editButton.blur();
+        if (this.editButton.current) {
+            this.editButton.current.blur();
         }
-        if (this.state.editMode && this.input) {
-            this.input.scrollLeft = this.input.scrollWidth;
-            this.input.focus();
+        if (this.state.editMode && this.input.current) {
+            this.input.current.scrollLeft = this.input.current.scrollWidth;
+            this.input.current.focus();
```

Comment:
I get an error:
- goto transaction overview
- click on the magnifying glass
- click on the edit button

But it checks for `this.input.current`, so I am not sure why this happens 🤔 

![Screen Shot 2021-12-01 at 6 03 58 PM](https://user-images.githubusercontent.com/546900/144279753-e72f855c-05e4-48c7-ace0-eac7658e3e0f.png)